clang/include/clang/Basic/Module.h

```diff
@@ -271,8 +271,22 @@ class alignas(8) Module {
     DirectoryEntryRef Entry;
   };
 
-  /// The headers that are part of this module.
-  SmallVector<Header, 2> Headers[5];
+private:
+  unsigned HeaderKindBeginIndex[6] = {};
```

Comment:
Why 6 rather than 5?

Reply:
actually, it should be `NumHeaderKinds` rather than a constant

Reply:
I used `NumHeaderKinds`, but explained the `+1` in a comment.